From 75c34bf04175c779f0ca3c91d29bde11311ab03d Mon Sep 17 00:00:00 2001
From: Rico Ganahl <rico.ganahl@bytesatwork.ch>
Date: Wed, 31 Jan 2024 10:57:19 +0100
Subject: [PATCH] bytesatwork: bawconfig: introduce ti am62px

Add hardware revision of bytes at work TI AM62Px SoM.

Signed-off-by: Rico Ganahl <rico.ganahl@bytesatwork.ch>
(cherry picked from commit 7a56df4a0d528a38c1a17a35cce9637999ec6fbf)
---
 board/bytesatwork/baw_config/baw_config.c | 2 ++
 board/bytesatwork/baw_config/baw_config.h | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/board/bytesatwork/baw_config/baw_config.c b/board/bytesatwork/baw_config/baw_config.c
index 70b2ef00cb..9c4429733e 100644
--- a/board/bytesatwork/baw_config/baw_config.c
+++ b/board/bytesatwork/baw_config/baw_config.c
@@ -19,6 +19,7 @@ const struct baw_config_name_pair baw_config_pcb_name[] = {
 	{ M8_PCB_REV_1_0, "M8 Rev 1.0" },
 	{ M8_PCB_REV_1_1, "M8 Rev 1.1" },
 	{ M7_PCB_REV_1_0, "M7 Rev 1.0" },
+	{ M8P_PCB_REV_1_0, "M8P Rev 1.0" },
 	{ -1, NULL },
 };
 
@@ -47,6 +48,7 @@ const struct baw_config_name_pair baw_config_ram_name[] = {
 	{ M7_RAM_MT53D512M32D2DS_053, "MT53D512M32D2DS-053 2 GB LPDDR4 @ 2000 MHz" },
 	{ M7_RAM_MT53E768M32D4DT_053, "MT53E768M32D4DT-053 3 GB LPDDR4 @ 2000 MHz" },
 	{ M7_RAM_MT53E1G32D2FW_046, "MT53E1G32D2FW-046 4 GB LPDDR4 @ 2000 MHz" },
+	{ M8P_RAM_MT53E2G32D4DE_046, "MT53E2G32D4DE-046 8 GB LPDDR4 @ 1600 MHz" },
 	{ -1, NULL },
 };
 
diff --git a/board/bytesatwork/baw_config/baw_config.h b/board/bytesatwork/baw_config/baw_config.h
index df7c7e1cd6..58d20b55cd 100644
--- a/board/bytesatwork/baw_config/baw_config.h
+++ b/board/bytesatwork/baw_config/baw_config.h
@@ -23,6 +23,7 @@ enum baw_config_pcb {
 	M8_PCB_REV_1_0 = 10,
 	M8_PCB_REV_1_1 = 11,
 	M7_PCB_REV_1_0 = 12,
+	M8P_PCB_REV_1_0 = 13,
 };
 
 enum baw_config_ram {
@@ -53,6 +54,7 @@ enum baw_config_ram {
 	M7_RAM_MT53D512M32D2DS_053 = 22,
 	M7_RAM_MT53E768M32D4DT_053 = 23,
 	M7_RAM_MT53E1G32D2FW_046 = 24,
+	M8P_RAM_MT53E2G32D4DE_046 = 25,
 };
 
 enum baw_config_flash {
-- 
2.39.2

